File: Python Excel Examples/RangesApiDemo/setRangeRowHeight.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.ranges_api.RangesApi(configuration)

name = "SetRangeRowHeight.xlsx"
folder = "Ranges"
storage = ""
sheetName = "Sheet1"
Range = spirecloudexcel.models.Range()
Range.first_row = 1
Range.first_column = 1
Range.row_count = 10
Range.column_count = 5
value = 20
api.set_range_row_height(name, sheetName, Range, value, storage = storage, folder = folder)

File: JavaScript Excel Examples/RangesApiDemo/setRangeRowHeight.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.RangesApi(apiClient);
    
    var name = "setRangeRowHeight.xlsx";
    var opts = { folder: "input" };
    var sheetName = "Sheet1";
    var range = new SpirecloudExcel.Range();
    range.FirstRow = 1;
    range.FirstColumn = 1;
    range.ColumnCount = 5;
    range.RowCount = 10;
    var value = 20;
    instance.setRangeRowHeight(name, sheetName, range, value, opts,
        function (error, data, response) {
        if (error) throw error;
    });
})();